ios/FluentUI/Core/SwiftUIAdapter.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+public struct SwiftUIAdapter {
```

Comment:
The goal here is to create a namespace, right? Shouldn't this be just an enum instead?

Reply:
Yes, the basic idea was to use namespace, I don't have any specific preference with enum vs struct for namespace.